qa/tasks/ceph.py

```diff
@@ -177,13 +177,12 @@ def invoke_logrotate(self):
             while not self.stop_event.is_set():
                 self.stop_event.wait(timeout=30)
                 try:
-                    run.wait(
-                        ctx.cluster.run(
-                            args=['sudo', 'logrotate', '/etc/logrotate.d/ceph-test.conf'
-                                  ],
-                            wait=False,
-                        )
```

Comment:
This part confuses me, `wait=False` yet why do we call `run.wait()`? 

However, the change itself looks good to me.

Reply:
I think because `run.wait` has nice debug output.

Reply:
Okay, thanks for the reply.